server/routes/support/handlers/offenderAudit.test.ts

```diff
@@ -7,7 +7,7 @@ import { AuditEvent } from '../../../@types/licenceApiClientTypes'
 import { convertToTitleCase } from '../../../utils/utils'
 
 const prisonerService = new PrisonerService(null, null) as jest.Mocked<PrisonerService>
-const licenceService = new LicenceService(null, null, null) as jest.Mocked<LicenceService>
+const licenceService = new LicenceService(null, null, null, null) as jest.Mocked<LicenceService>
```

Comment:
move all services into test service factory to prevent these cascading changes when dependencies change